omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java

```diff
@@ -39,9 +39,8 @@ public void controller(FragmentConfiguration config,
 		Object patient = config.get("patient");
 		Object visitType = config.get("visitType");
 
-		if(patient == null) {
+		if( (patient == null) && (request.getParameter("patientId") != null) ) {
```

Comment:
![image](https://user-images.githubusercontent.com/9057882/92788024-e4b04000-f3a9-11ea-8cfb-05708ba56ff6.png)
You have made unnecessary spaces in the places marked in red. Additionally, it is good style to make a space between the "if" and the opening brace.

Reply:
Thanks for the correction [Lukasz Debicki](https://github.com/sacull)
I have effected the changes